config/frontend.properties

```diff
@@ -12,6 +12,9 @@
 
 # rest server
 rest.server.blob.storage.service.factory=com.github.ambry.frontend.AmbryBlobStorageServiceFactory
+rest.server.router.factory=com.github.ambry.router.NonBlockingRouterFactory
```

Comment:
don't think this change should be made. There are many accompanying changes to be made (like setting success targets etc) to actually get this work (the quick start uses these properties). 

When we are ready to make this change, we will make the `NonBlockingRouter` the default.